### PR TITLE
Feat: text input right label

### PR DESCRIPTION
**Pull request by doichev-kostia**
[Ticket](https://github.com/Panenco/panenco-ui/projects/1#card-76402577]
Issue #211

### CODE REVIEW

**Comment by doichev-kostia** [comment]
@vlacher12

File: src/components/form/text-input/index.tsx [hunk 1]
```diff
@@ -61,28 +64,41 @@ export const TextInput = React.forwardRef<HTMLDivElement, TextInputProps>(
...
-            {title}
-          </Text>
-        )}
-        {subTitle && (
-          <Text size={theme.typography.sizes.xs} className="inputSubtitle">
-            {subTitle}
-          </Text>
-        )}
+        <div className='titleContainer'>
+          <div className={`${subtitle ? 'mb-4' : ''}`}>
```
Comment (vlacher12) [inline src/components/form/text-input/index.tsx:68]: just add `margin-bottom: 4px` to `titleContainer` class

Comment (doichev-kostia) [reply src/components/form/text-input/index.tsx:68]: In fact, margin-bottom should be a temporary property that depends on the availability of subtitle

Comment (lazzystep) [reply src/components/form/text-input/index.tsx:68]: @doichev-kostia 

```
<div className=`titleContainer ${subtitle ? 'mb-4' : ''}`>
```

File: src/components/form/text-input/index.tsx [hunk 2]
```diff
@@ -61,28 +64,41 @@ export const TextInput = React.forwardRef<HTMLDivElement, TextInputProps>(
...
+
+            {
+              subtitle && (
+                <Text size={theme.typography.sizes.xs} className='inputSubtitle inputSubtitleLeft'>
+                  {subtitle}
+                </Text>
+              )
+            }
+          </div>
+          <div className='mb-4'>
```
Comment (lazzystep) [inline src/components/form/text-input/index.tsx:83]: do we need empty `div` in case where `rightSubtitle` is missing?
@doichev-kostia

**Review by lazzystep — APPROVED** [review]

File: src/components/form/text-input/index.tsx (hunk 1 above)
Comment (doichev-kostia) [reply src/components/form/text-input/index.tsx:68]: @lazzystep, if I write template string without curly braces, I'll catch a syntax error

Comment (lazzystep) [reply src/components/form/text-input/index.tsx:68]: Yeah, agree, add them
you get the point about Vlad's message

File: src/components/form/text-input/index.tsx (hunk 2 above)
Comment (doichev-kostia) [reply src/components/form/text-input/index.tsx:83]: Yep, otherwise the layout will be displayed badly

Comment (lazzystep) [reply src/components/form/text-input/index.tsx:83]: Can it be redone somehow without empty div? @doichev-kostia 

divs allow flow content and empty divs are something generally to be avoided. You can read more [here](https://html.spec.whatwg.org/multipage/dom.html#palpable-content) about it.

**Comment by vlacher12** [comment]
1. 
``<div className={`${subtitle ? 'mb-4' : ''}`}>``
is not needed

2.  Rewrite
```
<div className='mb-4'>
            {
              rightSubtitle && rightSubtitle
            }
          </div>
```
by 

`{rightSubtitle && <div className="mb-4">{rightSubtitle}</div>}`

3. Add `mb-4` for both subTitle and rightSubtitle wrappers

**Comment by vlacher12** [comment]
@doichev-kostia

File: src/components/form/text-input/index.tsx (hunk 2 above)
Comment (doichev-kostia) [reply src/components/form/text-input/index.tsx]: @lazzystep, Thanks for an article)
I've fixed the layout issue :)

**Review by vlacher12 — APPROVED** [review]

**Pull request merged**